### ISSUE TITLE
a11y: improve Due Date field

### ISSUE THREAD

**Issue by candu** [CityofToronto/bdit_flashcrow#633]
**Description**
The "Due Date" field in our Request Study flows (both bulk and non-bulk, both create and edit) has a number of a11y shortcomings - this task addresses those.

**Acceptance Criteria**
- [x] the "Due Date" label should make the expected format clear (e.g. "Due Date: MM/DD/YYYY");
- [x] dropdown calendar should be in tab order.
